ui/ducks/domains.js

```diff
@@ -177,27 +253,46 @@ export function lookupEnsName(domainName) {
       ) &&
       !isHexString(trimmedDomainName)
     ) {
-      await dispatch(ensNotSupported());
+      await dispatch(domainNotSupported());
     } else {
-      log.info(`ENS attempting to resolve name: ${trimmedDomainName}`);
+      log.info(`Resolvers attempting to resolve name: ${trimmedDomainName}`);
       let address;
+      let fetchedResolutions;
+      let hasSnapResolution = false;
       let error;
       try {
-        address = await web3Provider.resolveName(trimmedDomainName);
+        address = await web3Provider?.resolveName(trimmedDomainName);
       } catch (err) {
         error = err;
       }
       const chainId = getCurrentChainId(state);
       const network = CHAIN_ID_TO_NETWORK_ID_MAP[chainId];
+      if (!address) {
+        // TODO: allow for conflict resolution in future iterations, we don't have designs
+        // for this currently, so just displaying the first result.
+        fetchedResolutions = await fetchResolutions({
+          domain: trimmedDomainName,
+          chainId: `eip155:${parseInt(chainId, 16)}`,
```

Comment:
Is the chain ID different here? We parse with radix 10 above

Reply:
radix 16 should be correct, we keep it as hex in state